Source/Scene/MetadataTileset.js

```diff
@@ -110,10 +103,9 @@ Object.defineProperties(MetadataTileset.prototype, {
   /**
    * Extensions in the JSON object.
    *
-   * @memberof MetadataClass.prototype
```

Comment:
Good catch!

Reply:
well I'll give `npm run build-ts` the credit for this one haha